test/bdd/pkg/didexchange/didexchange_controller_steps.go

```diff
@@ -29,6 +24,7 @@ import (
 	"github.com/hyperledger/aries-framework-go/pkg/doc/jose"
 	"github.com/hyperledger/aries-framework-go/test/bdd/pkg/context"
 	"github.com/hyperledger/aries-framework-go/test/bdd/pkg/sidetree"
+	"github.com/hyperledger/aries-framework-go/test/bdd/pkg/util"
```

Comment:
I don’t usually like a generically named util package - sometimes unrelated functions start appearing in them.

Reply:
Totally agree about names :)